docs/contents/user-docs/authorization.md

```diff
@@ -150,4 +150,74 @@ Some services may need an additional inline policy, specified in
 
 ## Cross-account resource management
 
-TODO
+ACK service controllers can manage resources in different AWS accounts. To enable and start using this feature, as an administrator, you will need to:
+1. Configure your AWS accounts, where the resources will be managed.
+2. Create a ConfigMap to map AWS accounts with the Role ARNs that needs to be assumed
+3. Annotate namespace with the AWS Account ID
+
+For detailed information about how ACK service controllers manage resource in multiple AWS accounts, please refer to [CARM](https://github.com/aws/aws-controllers-k8s/blob/main/docs/design/proposals/carm/cross-account-resource-management.md) design document.
+
+### Setting up AWS accounts
+
+AWS Account administrators should create/configure IAM roles to allow ACK service controllers to assume Roles in different AWS accounts.
+
+For example, to allow account A (000000000000) to create s3 buckets in account B (111111111111) you can use the following commands
+```bash
+# Using account B credentials
+aws iam create-role --role-name s3FullAccess \
+  --assume-role-policy-document '{"Version": "2012-10-17","Statement": [{ "Effect": "Allow", "Principal": {"AWS": "arn:aws:iam::000000000000:root"}, "Action": "sts:AssumeRole"}]}'
+aws iam attach-role-policy --role-name s3FullAccess \
+  --policy-arn 'arn:aws:iam::aws:policy/service-role/AmazonS3FullAccess'
+```
+
+### Map AWS Accounts with their associated Role ARNs
+
+After you will need to create a ConfigMap to associate each AWS Account ID with the role ARN that needs be assumed, in order to manage resource in that particular account.
```

Comment:
Please encapsulate `ConfigMap` with backticks here and throughout. Same with `Namespace` (which should be capitalized too)

Reply:
s/manage resource in/manage resources in/

Reply:
Done